### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jhslim**
Adding changes up to lesson 7 activity

### CODE REVIEW

File: blogging/admin.py [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
-# Register your models here.
 from blogging.models import Post, Category
 
-admin.site.register(Post)
-admin.site.register(Category)
\ No newline at end of file
+
+class CategoryInPostInline(admin.TabularInline):
+    model = Category.posts.through
+    extra = 2
```
Comment (ldconejo) [inline blogging/admin.py:7]: You don't need line 7.

File: blogging/admin.py [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+
+
+class PostAdmin(admin.ModelAdmin):
```
Comment (ldconejo) [inline blogging/admin.py:10]: Use the decorator to register _PostAdmin_ as _Post_:

```
@admin.register(Post)
class PostAdmin(admin.ModelAdmin):
    inlines = [CategoryInline]
```

File: blogging/admin.py [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+    inlines = [
+        CategoryInPostInline,
+    ]
+
+
+class CategoryAdmin(admin.ModelAdmin):
```
Comment (ldconejo) [inline blogging/admin.py:16]: Same here with the decorator:

```
@admin.register(Category)
class CategoryAdmin(admin.ModelAdmin):
    exclude = ('posts',)
```

File: blogging/admin.py [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
+    inlines = [
```
Comment (ldconejo) [inline blogging/admin.py:17]: You don't need inlines in the category admin, you're just removing posts from it, the exclude statement does that.

File: blogging/admin.py [hunk 5]
```diff
@@ -1,7 +1,23 @@
...
+        CategoryInPostInline,
+    ]
+    exclude = ('posts',)
+
+admin.site.register(Post, PostAdmin)
```
Comment (ldconejo) [inline blogging/admin.py:22]: No need for lines 22 and 23, the decorators will take care of that.

**Comment by ldconejo** [comment]
Hi Josh,

See my suggestions in your code, you're very close, the suggested changes should get you up and running.

Happy coding,

Luis

**Comment by jhslim** [comment]
Thanks Luis! I've got it working. I've submitted a new PR:
https://github.com/jhslim/django-blog/pull/4

Please use the one above for assignment 7.

Thanks,
Josh

**Comment by ldconejo** [comment]
Awesome! Works as expected now.

-Luis

**Pull request closed without merging**